src/App.tsx

```diff
@@ -1,6 +1,8 @@
 import {Container, makeStyles} from '@material-ui/core';
 import React from 'react';
-import {BrowserRouter, Route} from 'react-router-dom';
+import {BrowserRouter} from 'react-router-dom';
+// @ts-ignore
```

Comment:
what was typescript doing here?

Reply:
So @type imports are needed, but an @type module doesn't exist for ReactRouterGlobalHistory. I had no choice but the add that.